data-collections-exercises.md

```diff
@@ -9,95 +9,99 @@ planeteers = ["Earth", "Wind", "Captain Planet", "Fire", "Water"]
 Access the second value in `planeteers`.
 
 ```rb
-# Your answer here
+planeteers[1]
 ```
 
 Add "Heart" to the end of `planeteers`.
 
 ```rb
-# Your answer here
+planeteers.push("heart")
 ```
 
 Remove "Captain Planet" from the array (without using a method).
 
 ```rb
-# Your answer here
+planeteers[2]=nil
```

Comment:
I'm getting what you're trying to do, but you still technically didn't remove `"Captain Planet"` your array will be like this ["Earth", "Wind", nil, "Fire", "Water"] , to remove it without using a method we can subtract the value like this  `planeteers - ["Captain Planet"]`